src/handlers/localstorage/globalSettings.js

```diff
@@ -63,3 +64,20 @@ export const saveTransactionSignatures = transactionSignatures =>
  */
 export const getTransactionSignatures = () =>
   getGlobal(TRANSACTION_SIGNATURES, {});
+
+/**
+ * @desc get dev enabled preference
+ * @param  {Boolean}  [value]
+ */
+export const getTestnetsEnabled = async preference => {
+  const test = getGlobal(TESTNETS_ENABLED, preference);
```

Comment:
the second param is the default value that we should return for when the key TESTNETS_ENABLED cannot be found (for instance, on the very first time a user reads this key before setting any future value)